### ISSUE TITLE
Add file reference as overriding parameters

### ISSUE THREAD

**Issue by goldstein0101** [databrickslabs/dbx#641]
Hey, one of the main features of dbx is having automated configuration with file refrences inside deployment conifguration.
Also i guess configuration files are a lot of times part as an parameters of task. And the only way to references configration files as parameters are inside deployment configuration.
But i think alot of people are using dbx launch, launch from assets, execute for development and triggering jobs where they want to override parameters where those parameters could be any file path inside project.
Will u going to make a feature that will take care of this?

**Comment by copdips** [comment]
I'm not from DBS but I cannot find where I can do it from the UI. If you do, could you please share it ?

**Comment by goldstein0101** [comment]
What do u mean from the UI?

My comment is that: when u specify any kind of file with: https://dbx.readthedocs.io/en/latest/features/file_references/,
it will be automatically converted into correct path, since every deployment means new mflow job id, hence new file structure under your deployment place on dbfs. Inside deployment.yaml, its done automatically, but if i want to override file paths as a CLI argument with dbx launch or execute or launch from assets, it will not take care of that.
So i would have to manually write some wrapper to get the latest deployment, get that id, and make a file path out of it.

**Comment by copdips** [comment]
sorry I misunderstood, for overriding, maybe we can:

```bash
overide locally, than
dbx deploy workflow_name --assets-only --tags "workflow=workflow_name" --tags "user=my_name"
dbx launch workflow_name --from-assets --tags "workflow=workflow_name" --tags "user=my_name"
```

**Comment by goldstein0101** [comment]
Yeah that its current solution, having static place for configuration =).
